Rakefile.rb

```diff
@@ -1,32 +1,21 @@
-# Rakefile
-
 require_relative 'test_data'
 
 namespace :docker do
-  desc "Download a Docker image and start a container"
-  task :pull_and_run_container do
-    version = TestData[:version] || 'latest'
+  desc "Download Docker image"
+  task :pull_image do
```

Comment:
You can name the command similarly to docker command names, as it follows the logic